src/App.js

```diff
@@ -2,6 +2,7 @@ import WelcomeScreen from './components/screens/welcome-screen'
 import LoginScreen from './components/screens/login-screen'
 import MainMenu from "./components/screens/main-menu";
 import SettingsScreen from "./components/screens/settings-screen";
+import SettingsSoundScreen from "./components/screens/settings-sound-screen";
```

Comment:
W takiej kolejności nazwa komponentu brzmi trochę dziwnie

```suggestion
import SoundSettingsScreen from "./components/screens/settings-sound-screen";
```